packages/calcite-components/src/components/input/input.tsx

```diff
@@ -679,6 +679,11 @@ export class Input
     if (this.disabled || this.readOnly) {
       return;
     }
+
+    if (this.type === "file") {
```

Comment:
It looks like we can remove the same logic from `inputChangeHandler` since input would always emit before change, right?